content/F2015/F15-event-dropbox-coffee.md

```diff
@@ -0,0 +1,18 @@
+title: Dropbox Coffee Chats
+Date: 2015-09-5
+Category: Events
+Tags: social, dropbox
```

Comment:
We have a BlackBerry tag but I'm guessing that's because we had a contract with them. ++ replacing the 'dropbox' tag with the 'industry' tag.